test/test-applications/instrumentation/CallTargetNativeTest/CallTargetNativeTest.csproj

```diff
@@ -10,4 +10,9 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
+    <ProjectReference Include="..\..\..\..\src\Datadog.Trace.ClrProfiler.Managed\Datadog.Trace.ClrProfiler.Managed.csproj" />
+  </ItemGroup>
+
```

Comment:
Yes, because it declares its own integrations: https://github.com/DataDog/dd-trace-dotnet/blob/master/test/test-applications/instrumentation/CallTargetNativeTest/NoOp/Noop0ArgumentsIntegration.cs